src/java.base/share/classes/java/lang/Runtime.java

```diff
@@ -45,10 +44,66 @@
  * {@code Runtime} that allows the application to interface with
  * the environment in which the application is running. The current
  * runtime can be obtained from the {@code getRuntime} method.
+ *
+ * <p>An application cannot create its own instance of this class.
+ *
+ * <h2><a id="shutdown">Shutdown Sequence</a></h2>
+ *
+ * <p>The Java Virtual Machine initiates the <i>shutdown sequence</i> in response
+ * to one of several events:
+ * <ol>
+ * <li>when the number of {@linkplain Thread#isAlive() live} non-daemon threads drops to zero
+ * for the first time;</li>
+ * <li>when the {@link #exit Runtime.exit} or {@link System#exit System.exit} method is called
+ * for the first time; or</li>
+ * <li>when some external event occurs, such as an interrupt or a signal is received from
+ * the operating system.</li>
+ * </ol>
+ *
+ * <p>At the beginning of the shutdown sequence, the registered shutdown hooks are
+ * {@linkplain Thread#start started} in some unspecified order. They run concurrently
+ * with any daemon or non-daemon threads that were {@linkplain Thread#isAlive() alive}
+ * at the beginning of the shutdown sequence.
+ *
+ * <p>After the shutdown sequence has begun, registration and de-registration of shutdown hooks
+ * with {@link #addShutdownHook addShutdownHook} and {@link #removeShutdownHook removeShutdownHook}
+ * is prohibited. However, creating and starting new threads is permitted. New threads run
+ * concurrently with the registered shutdown hooks and with any daemon or non-daemon threads
+ * that are already running.
+ *
+ * <p>The shutdown sequence finishes when all shutdown hooks have terminated. At this point,
+ * the Java Virtual Machine terminates as described below.
+ *
+ * <p>It is possible that one or more shutdown hooks do not terminate, for example, because
+ * of an infinite loop. In this case, the shutdown sequence will never finish. Other threads
+ * and shutdown hooks continue to run and can terminate the JVM via the {@link #halt halt} method.
+ *
+ * <p>Prior to the beginning of the shutdown sequence, it is possible for a program to start
+ * a shutdown hook by calling its {@link Thread#start start} method explicitly. If this occurs, the
+ * effect on the shutdown sequence is unspecified.
+ *
+ * <h2><a id="termination">Virtual Machine Termination</a></h2>
```

Comment:
Should this say "Java Virtual Machine Termination" for consistency.

Reply:
will fix